tests/transactional/host_config.pm

```diff
@@ -14,15 +14,18 @@ use strict;
 use warnings;
 use testapi;
 use transactional qw(process_reboot);
+use bootloader_setup qw(change_grub_config);
 
 sub run {
     select_console 'root-console';
 
     # GRUB Configuration
     my $disable_grub_timeout = get_var('DISABLE_GRUB_TIMEOUT');
     my $extrabootparams = get_var('EXTRABOOTPARAMS');
-    assert_script_run("sed -i 's/GRUB_TIMEOUT=.*/GRUB_TIMEOUT=-1/' /etc/default/grub") if $disable_grub_timeout;
-    assert_script_run("sed -i 's/^GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*/& $extrabootparams/' /etc/default/grub") if $extrabootparams;
+    change_grub_config('=\"[^\"]*', "& $extrabootparams", 'GRUB_CMDLINE_LINUX_DEFAULT') if $extrabootparams;
+    change_grub_config('=.*', '=-1', 'GRUB_TIMEOUT') if $disable_grub_timeout;
+    change_grub_config('=.*', '=1024x768', 'GRUB_GFXMODE');
+    change_grub_config('^#', '', 'GRUB_GFXMODE');
```

Comment:
I thought that setting gfx mode will be helpful and aligned the behaviour with the installation. It was wrong, grub2 resolution is not full screen in DVD installations. I can either remove it, or we can keep it as it looks nicer and yet it is a valid test. But no objections if we go for dropping it.